tensorflow/lite/core/api/flatbuffer_conversions.cc

```diff
@@ -2200,6 +2200,14 @@ TfLiteStatus ParseSquare(const Operator*, ErrorReporter*, BuiltinDataAllocator*,
   return kTfLiteOk;
 }
 
+// We have this parse function instead of directly returning kTfLiteOk from the
+// switch-case in ParseOpData because this function is used as part of the
+// selective registration for the OpResolver implementation in micro.
+TfLiteStatus ParseSquaredDifference(const Operator*, ErrorReporter*, BuiltinDataAllocator*,
```

Comment:
Thanks for adding this change! It looks good to me and we do want all the changes here. However, `flatbuffer_conversions.cc/h` are files that we pull from TFLite, so those changes will have to go into TFLite first. The process is a little involved, and currently the TFLite->TFLM sync is broken, so we will handle the changes in these two files here. We'll revisit this PR after the sync is completed, which means this PR should then only contain changes in the `micro` folder.

Reply:
@mikljohansson The changes in `flatbuffer_conversions.cc/h` have been sync'd to the TFLM repo, please remove the changes here and we can proceed to merge this PR.